CHANGELOG.rst

```diff
@@ -2,6 +2,120 @@
 Change log for Preview-generator
 ================================
 
+----------
+0.27 / TBD
+----------
+
+:warning: This release introduce many changes with two of the basic builders changed (image, pdf) and a new one added (raw).
+
+Features
+~~~~~~~~
+
+- deprecate Pillow and Imagemagick command line builder in favor of a new Wand builder #253
+- add Webp format support to the new Wand builder #273
+- replace the old pdf builder by a new poppler-utils based builder #192
+- add a new builder for raw using rawpy
+
+-----------------
+0.26 / 2021-11-16
+-----------------
+
+Fixed issues
+~~~~~~~~~~~~
+
+- enforce office mimetype supported by libreoffice to avoid unknown mimetype issues related to system specific configuration:  #283
```

Comment:
```suggestion
- enforce office mimetypes supported by LibreOffice to avoid unknown mimetype issues related to system specific configuration:  #283
```